### PR TITLE
update bio for 2016

### PR DESCRIPTION
**Pull request by jcarbaugh**

### CODE REVIEW

File: bio.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+Jeremy Carbaugh is a Director of Technology. With over 15 years of experience building for the web, Jeremy has created everything from rapid campaigns for scrappy nonprofits to large enterprise applications for the FBI. He helps the team build scalable architectures and elegant systems that are well tested and secure. He is an avid open source contributor, active in the DC tech and Python communities, and a two-time Webby honoree.
+
+Jeremy loves code, especially when it helps make people's jobs easier and lives better.
```
Comment (jcarbaugh) [inline bio.md]: This sounds okay to me.

**Comment by paultag** [comment]
:confetti_ball:

File: bio.md (hunk 1 above)
Comment (paultag) [inline bio.md]: approved

**Comment by jcarbaugh** [comment]
@paultag: thanks for participating in this demo!

**Comment by paultag** [comment]
@jcarbaugh Here's a comment, look, it updates in real time!

**Pull request merged**